tests/monotouch-test/MetalPerformanceShaders/KernelTest.cs

```diff
@@ -3,43 +3,90 @@
 #if !__WATCHOS__
 
 using System;
+using Foundation;
 
 #if XAMCORE_2_0
 using Metal;
 using MetalPerformanceShaders;
-using UIKit;
 #else
 using MonoTouch.Metal;
 using MonoTouch.MetalPerformanceShaders;
-using MonoTouch.UIKit;
 #endif
 
 using NUnit.Framework;
 
-namespace MonoTouchFixtures.MetalPerformanceShaders {
+namespace MonoTouchFixtures.MetalPerformanceShaders
+{
 
 	[TestFixture]
-	public class KernelTest {
+	public class KernelTest
+	{
 
 		[Test]
-		public void RectNoClip ()
+		public void RectNoClip()
 		{
+#if !MONOMAC
 			TestRuntime.AssertXcodeVersion (7,0);
+#else
+			TestRuntime.AssertXcodeVersion(9, 0);
+#endif
 
 			var d = MTLDevice.SystemDefault;
 			// some older hardware won't have a default
 			if (d == null)
-				Assert.Inconclusive ("Metal is not supported");
-			
+				Assert.Inconclusive("Metal is not supported");
```

Comment:
There are still missing spaces :smile: and in this case it's also diff noise, since the only change is the removed space.